tox.ini

```diff
@@ -20,10 +20,26 @@ commands = nosetests --ignore-files=test_appengine\.py {posargs}
 [testenv:cover]
 basepython = python2.7
 commands =
-    nosetests --with-xunit --with-xcoverage --cover-package=oauth2client --nocapture --cover-erase --cover-tests --cover-branches --ignore-files=test_appengine\.py --cover-min-percentage=60
+    nosetests \
```

Comment:
confirmed, I can add

```
coverage report --fail-under 80
```

To the end of this to make it fail if the combined coverage is under 80. Do you want me to do that?